articles/flows/guides/m2m-flow/call-api-using-m2m-flow.md

```diff
@@ -17,24 +17,20 @@ useCase:
 This tutorial will help you implement the M2M flow. If you want to learn how the flow works and why you should use it, see [M2M Flow](/flows/concepts/m2m-flow).
 :::
 
-Auth0 makes it easy for your app to implement the M2M flow using:
-
-* [Auth0 SDKs](/libraries): The easiest way to implement the M2M flow, which will do most of the heavy-lifting for you. Our [Backend/API Quickstarts](/quickstart/backend) will walk you through the process.
-* Authentication API: If you prefer to roll your own, keep reading to learn how to call our API directly.
+Auth0 makes it easy for your app to implement the M2M flow.
 
 Following successful authentication, the calling application will have access to an [Access Token](/tokens/overview-access-tokens), which can be used to call your protected APIs.
 
 ## Prerequisites
 
 **Before beginning this tutorial:**
 
+* [Register your API with Auth0](/architecture-scenarios/server-api/part-2#configure-the-api)
```

Comment:
If we are going to talk about scopes in Call an API now, then we should probably create some scopes for this API, so there is something to select in the "Register M2M App" section below.

Reply:
Agreed, if we're going to use scopes in the doc, at very minimum, this item should say Register your API and create scopes. If we want to go farther we could give some specific instructions or link to another doc, but at minimum we should at least _say_ that you need to set up some scopes.

Reply:
**From Andres document:**
When you create a M2M app, **you can’t select the application type**, the **grant types always include Client Credentials**, and it **forces you to select an API during the creating flow.**

Steps would be:

Register your API
Register the M2M application with Auth0, authorizing the your API

Reply:
Removing the mention of scopes from the ` * Choose your previously registered API`  line I think satisfied this?